### ISSUE TITLE
Most Codecs Invalid?

### ISSUE THREAD

**Issue by armadillu** [elliotwoods/ofxSquash#8]
Hey, I'm trying to make the simplest example run on OS X 10.10 and Xcode 7.3.

First thing is that I need to have 2 copies of each plugin dylib - if the dylibs are not present in "bin/" the app refuses to launch (dyld image not found). This is ok, but not sure if you came across this - the example doesn't seem to have any scripting to do anything about it (i.e. trying to get the binary to load the dylibs from the plugin folder somehow)

Once I get the app to run (by copying each plugin dylib into the bin/ folder), I see only one codec is listed as valid; the "brieflz" codec... And then if I try to use it, the app crashes here:

![screen shot 2016-04-22 at 13 09 20](https://cloud.githubusercontent.com/assets/167057/14749088/73ba743c-088b-11e6-9408-c179a3bc2d04.PNG)

Am I missing something, or something is off with my setup?

**Comment by elliotwoods** [comment]
are the plugins in bin/plugins/squash?

are you running x64?

**Comment by armadillu** [comment]
Yep to both. Plugins are found - if they are not in the right place, the addons prints a message about it.
